libs/designsystem/src/lib/components/progress-circle/progress-circle.component.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  Component,
+  Input,
+  ChangeDetectionStrategy,
+  HostBinding,
+  ElementRef,
+  AfterViewInit,
+  ChangeDetectorRef,
+  OnDestroy,
+} from '@angular/core';
+
+const RADIUS_MAP = {
+  sm: 20,
+  md: 28,
+  lg: 48,
+};
+
```

Comment:
Consider swapping out with a `DIAMETER_MAP` as it's more in line with UX sizes:
```suggestion
const DIAMETER_MAP = {
  sm: 40,
  md: 56,
  lg: 96,
};

```

Reply:
Changed to diameter map